client/src/containers/Economy/components/GDP/index.tsx

```diff
@@ -19,6 +19,26 @@ export const GDP = ({ onData }) => {
     });
   };
 
+  const changeNumberFormat = (val) => {
```

Comment:
Move this function to the separate file. It may be used for other places where numbers are huge.